File: ORTL1P04 - Average/Average.R
# Project name : SPOJ: ORTL1P04 - Average
# Author       : Wojciech Raszka
# Date created : 2019-04-02
# Description  :
# Status       : Accepted (23554130)
# Tags         : R
# Comment      : 100 score

f <- file('stdin', open='r')

tokens <- as.integer(unlist(strsplit(readLines(f, n=1), " ")))

m <- mean(tokens)
write(m, stdout())
if (m >= 6){
  write("APROBADO", stdout())
} else {
  write("DESAPROBADO", stdout())
}

File: AVG - AVERAGE/AVERAGE.py
# Project name : SPOJ: AVG - AVERAGE
# Author       : Wojciech Raszka
# E-mail       : gitpistachio@gmail.com
# Date created : 2019-04-14
# Description  :
# Status       : Accepted (23634195)
# Tags         : python, math, average
# Comment      :

T = int(raw_input())

while T > 0:
	n = int(raw_input())
	print sum(map(int, raw_input().split()))/n
	T -= 1
